Libplanet/Action/ActionEvaluator.cs

```diff
@@ -439,11 +440,16 @@ ActionContext CreateActionContext(IAccountStateDelta prevStates, int randomSeed)
 
                 // FIXME: This is dependant on when the returned value is enumerated.
                 TimeSpan evalDuration = DateTimeOffset.Now - startTime;
+                const string TimestampFormat = "yyyy-MM-ddTHH:mm:ss.ffffffZ";
                 _logger
                     .ForContext("Tag", "Metric")
                     .Debug(
-                        "Actions in transaction {TxId} evaluated in {DurationMs:F0}ms.",
+                        "Actions in transaction {TxId} by {Signer} with timestamp {TxTimestamp} " +
```

Comment:
It might be better if the number of actions is also printed.

Reply:
Done. See c5d076f4243150d82df987018c7f36a9759f9ec9.